.github/workflows/sub_ui.yaml

```diff
@@ -163,7 +163,15 @@ jobs:
 
       - name: Install Chartmuseum
         id: install_chartmuseum
-        run: cd tests && make e2e-install-chartmuseum
+        run: |
+          cd tests && make e2e-install-chartmuseum
+          # Extract latest dev operator version
+          OPERATOR_DEV_CHART=$(ls elemental-operator-chart*)
```

Comment:
I would have add a `-` before the `*`, even if I'm not sure that we may have something like:
````bash
$ ls
elemental-operator-chart-1.5.0.tgz elemental-operator-charts-1.6.0.tgz
```